assets/src/edit-story/units/dimensions.js

```diff
@@ -0,0 +1,39 @@
+
+/**
+ * Internal dependencies
+ */
+import { PAGE_WIDTH, PAGE_HEIGHT } from '../constants';
+
+export function dataUnit( v ) {
+	return Number( v.toFixed( 0 ) );
+}
+
+export function editorUnit( v ) {
+	return Number( v.toFixed( 5 ) );
+}
+
+export function dataToEditorX( x, pageWidth ) {
+	return editorUnit( x * pageWidth / PAGE_WIDTH );
+}
+
+export function dataToEditorY( y, pageHeight ) {
+	return editorUnit( y * pageHeight / PAGE_HEIGHT );
+}
+
+export function editorToDataX( x, pageWidth ) {
+	return dataUnit( x * PAGE_WIDTH / pageWidth );
+}
+
+export function editorToDataY( y, pageHeight ) {
+	return dataUnit( y * PAGE_HEIGHT / pageHeight );
+}
+
+export function getBox( { x, y, width, height, rotationAngle, isFullbleed }, pageWidth, pageHeight ) {
+	return {
+		x: dataToEditorX( isFullbleed ? 0 : x, pageWidth ),
+		y: dataToEditorY( isFullbleed ? 0 : y, pageHeight ),
```

Comment:
I wonder if it would make sense to use the percentage values directly (storing data in % instead of px as now) and instead do the conversion in places before saving and displaying the data instead where necessary (e.g. in the Size panel and Moveable). Thoughts?

Reply:
I think we've did some analysis for this before and it seemed like we may want to arrive at an integer value at some point and hence we decided to pick a predefined data pixel. I still think it's easier. For instance, `Moveable` then uses the same coordinate system as we. While we could try to define our position in `%`, `Moveable` still measures and uses pixel values that we then compare, add, etc. If you'd like to dig deeper, should we reopen https://github.com/google/web-stories-wp/issues/71?

Reply:
Pixels definitely made sense with the idea to keep the width/height for Page the same all the time. Yes, `Moveable` would be one place where we'd need to convert between `%` and `px` which I suppose, would still require responsive converting anyway.

Perhaps it's safer to use `px` and if you think it's still easier then let's stick to that. Was just wondering for a moment if now, when the Page will be responsive, `%` might make sense in the editor, too. However, to prevent struggle and potential future surprises with `Moveable` and possible other libs then `px` can still be a better option.

Reply:
Let's continue this thread in that issue. This PR makes this a bit more systematic. So it should help in migration if we decide to go down that path.